person.rb

```diff
@@ -0,0 +1,30 @@
+require 'nameable'
+
+class Person < Nameable
+  attr_accessor :name, :age
+  attr_reader :id, :rental
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    super()
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    @rental = []
+  end
+
```

Comment:
- [ ] A method for adding rentals should be present, and the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association 🙃 
It can be similar to 👇🏾 

```ruby
 def add_rental(book, date)
    Rental.new(date, book, self)
  end
```